appeals/web/src/server/lib/mappers/appellant-case/appeal-type-s78.mapper.js

```diff
@@ -0,0 +1,71 @@
+import { removeSummaryListActions } from '#lib/mappers/mapper-utilities.js';
```

Comment:
Breaking it down by appeal type is a great idea! Love it!

Reply:
Thanks! I've renamed this file to `appeal-type-fpa.mapper.js` now as discussed, to avoid using the S78 terminology.